### PR TITLE
Reorganise AWS docs and Apps tab

### PR DESCRIPTION
**Pull request by thesollyz**
This PR rewrites and reorganises the AWS docs and the Apps tab:
- Create a central place for all data collection from AWS services (Send Data > AWS).
- Clearly separate Send Data and Apps sections.
- Update information about collecting data from different AWS services based on GitHub readmes.
- Restructure many existing pages to better fit current style.
- Move PagerDuty to Monitors section.
- Add new pages:
  - Overview page for AWS services
  - Send data from Firehose
  - Use AWS Lambda app to enrich your Axiom organization

This PR is a prerequisite for changing the GitHub readmes:
- https://github.com/axiomhq/axiom-lambda-extension/pull/32
- https://github.com/axiomhq/axiom-cloudwatch-forwarder/pull/84
- https://github.com/axiomhq/axiom-cloudfront-lambda/pull/21

### CODE REVIEW

File: send-data/aws-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| CodeBuild                                            | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| CodeWhisperer event logs                             | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Cognito                                              | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Connect                                              | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| DataSync                                             | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| ECS                                                  | [Fluentbit](/send-data/fluent-bit)                                                                                                   |
```
Comment (manototh) [inline send-data/aws-overview.mdx]: Is this true? Do ECS users need to use Fluentbit?

File: send-data/aws-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| API Gateway                                          | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| AppSync                                              | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Aurora MySQL                                         | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Beanstalk                                            | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Bedrock Knowledge                                    | [AWS Cloudwatch Forwarder](/send-data/cloudwatch), [AWS S3 Forwarder](/send-data/aws-s3) or [AWS Firehose](/send-data/aws-firehose)  |
+| Chime media quality metric logs and SIP message logs | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
```
Comment (manototh) [inline send-data/aws-overview.mdx]: Can we keep this simpler? Just say Chime and create another row for SIP?

File: send-data/aws-overview.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+| Cloudfront                                           | [AWS S3 Forwarder](/send-data/aws-s3)                                                                                                |
+| CloudHSM audit logs                                  | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Cloudtrail                                           | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Cloudwatch                                           | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| CloudWatch (evidently evaluation event logs)         | [AWS Cloudwatch Forwarder](/send-data/cloudwatch) or [AWS Firehose](/send-data/aws-firehose)                                         |
```
Comment (manototh) [inline send-data/aws-overview.mdx]: `evidently evaluation event logs` do we need to say this here? Can this be highlighted in the docs on Cloudwatch Forwarder?

**Comment by manototh** [comment]
@schehata Thanks a lot for this. I've restructured and rewritten it to align it better with the rest of the docs.

One thing I've noticed is that the table you'd created references a docs page on Firehose which we don't have yet. Do we even support this?

**Comment by dominicchapman** [comment]
@tothmano this is really taking shape. Great job.

Some comments:
- Amazon CloudWatch, not AWS CloudWatch. This is widespread in material, so let’s please update all references.
- We intentionally named the CloudWatch Forwarder as Axiom CloudWatch Forwarder to make it clear that we were responsible for creating this tooling. When I read these docs, it would be easy for me to think that Amazon were responsible for the CloudWatch Forwarder, which they are not. I expect the name change will be sufficient to clear that up. 
- We should simultaneously make PRs to clean up the descriptions from AWS related GitHub repos like https://github.com/axiomhq/axiom-cloudwatch-forwarder with a pointer back to our docs instead.
- Knowing that the Axiom CloudWatch Forwarder (and other collection methods where this is true) is open source is important. We should include some kind of note on the associated docs page that source code is open source, available at [link] and welcome improvements via contributions.
- Let’s invite people to submit requests for support for other AWS services with a callout block or similar. It could encourage readers to contact us at support@axiom.co with requests for new AWS services.

Fantastic AWS Lambda Apps page. This is exactly what I was hoping for. In a future iteration, it would be delightful to include a screenshot or similar to bring that experience to life. No need to stall this work on that, though.

Hope this helps!

**Review by dominicchapman — APPROVED** [review]
Great job!

**Pull request merged**